pkg/ipxe/Dockerfile

```diff
@@ -16,8 +17,10 @@ RUN cat /tmp/*patch | patch -p1
 # bin/ipxe.iso
 ENV TARGET_x86_64  bin-x86_64-efi/ipxe.efi bin/ipxe.dsk bin/ipxe.lkrn bin/undionly.kpxe
 ENV TARGET_aarch64 bin-arm64-efi/ipxe.efi
+ENV TARGET_riscv64 clean
```

Comment:
this is a bit unexpected. Why targets for x86_64 and aarch64 are building staff, and riscv64 is quite the opposite?

Reply:
B/c on riscv64 we need an empty packages (since ipxe hasn't really be ported to riscv64 just yet)

Reply:
makes a lot of sense. I would add short line of comment here for future myself (e.g. not ported yet)